packages/modeling/src/geometries/path2/appendArc.js

```diff
@@ -36,11 +36,10 @@ export const appendArc = (options, geometry) => {
 
   // validate the given options
   if (!Array.isArray(endpoint)) throw new Error('endpoint must be an array of X and Y values')
-  if (endpoint.length < 2) throw new Error('endpoint must contain X and Y values')
-  endpoint = vec2.clone(endpoint)
+  if (endpoint.length !== 2) throw new Error('endpoint must contain X and Y values')
```

Comment:
I was trying to remove unnecessary use of `clone`. By removing the call to `vec2.clone`, it subtly changes the behavior of this `appendArc` function. If you passed in a `vec3` instead of a `vec2`, previously the clone would have just taken the X and Y and dropped the Z value. By removing the clone, it would persist the `vec3`. This feels weird to me that we allow a `vec3` to be passed in for a 2D arc, so I changed the length check to be strictly equal to 2.